include/yaclib/util/result.hpp

```diff
@@ -118,6 +121,19 @@ class Result {
     }
   }
 
+  CrefT<V> Ok() & {
```

Comment:
```suggestion
  ConstValueRef Ok() const& {
```
Also plz, use MoveValue for `Ok() &&`